growth/too/gcn.py

```diff
@@ -171,7 +171,9 @@ def get_skymap(gcn_notice, root):
     gcn.NoticeType.LVC_RETRACTION,
     gcn.NoticeType.AMON_ICECUBE_COINC,
     gcn.NoticeType.AMON_ICECUBE_HESE,
-    gcn.NoticeType.AMON_ICECUBE_EHE
+    gcn.NoticeType.AMON_ICECUBE_EHE,
```

Comment:
Do we still want to listen to EHE alerts?

Reply:
Yeah, removed it